src/style.css

```diff
@@ -0,0 +1,98 @@
+:root {
+  --avenir-family: "Avenir";
```

Comment:
It is better to name variables not by content, but by value, because the content can change during the development process, so it would be better to name it `--main-font-family`